compiler-plugin-test/src/test/resources/codeaction/mandatory-relation-entity-optional-type.bal

```diff
@@ -51,13 +50,14 @@ type Building3 record {|
     string country;
     string postalCode;
     Workspace3? workspace2;
+	Workspace3 workspace3;
```

Comment:
```suggestion
    Workspace3? workspace2;
    Workspace3 workspace3;
```

Reply:
This is formatted due to the added tabspace. When inserting the row, I am adding a tabspace before hand. Should this be changed to 4 spaces?